app-web/__tests__/gatsby/getFilesFromRegistry.test.js

```diff
@@ -27,52 +28,6 @@ const SOURCE_3 = {
   },
 };
 
-const registry = [
```

Comment:
lets move the registry into a fixture file if it already hasn't been done

Reply:
It has been moved to __fixtures__/fixtures.js